src/realm/sync/subscriptions.cpp

```diff
@@ -698,11 +654,11 @@ std::pair<int64_t, int64_t> SubscriptionStore::get_active_and_latest_versions()
     descriptor_ordering.append_sort(SortDescriptor{{{sub_sets->get_primary_key_column()}}, {false}});
     descriptor_ordering.append_limit(LimitDescriptor{1});
     auto res = sub_sets->where()
-                   .equal(m_sub_set_keys->state, static_cast<int64_t>(SubscriptionSet::State::Complete))
+                   .equal(m_sub_set_state, static_cast<int64_t>(SubscriptionSet::State::Complete))
                    .find_all(descriptor_ordering);
 
     if (res.is_empty()) {
-        return {0, latest_id};
+        return {-1, latest_id};
```

Comment:
you return 0 above. any reason it was changed to -1? (or more like, is it needed?)